src/app/containers/WalletProvider/components/TxRequestDialog.tsx

```diff
@@ -73,21 +88,21 @@ export function TxRequestDialog({ open, type, amount, asset, error }: Props) {
       </Dialog>
     </>
   );
-}
+};
 
 const WLContainer = styled.div`
-  width: 98px;
-  height: 98px;
+  width: 160px;
+  height: 160px;
   border-radius: 1.25rem;
   border: 1px solid #e8e8e8;
-  margin: 0 auto 35px;
+  margin: 0 auto 1rem;
   div {
     font-size: 0.75rem;
   }
 `;
 const WLImage = styled.img`
-  width: 50px;
-  height: 50px;
+  width: 85px;
+  height: 85px;
   margin-bottom: 10px;
   object-fit: contain;
 `;
```

Comment:
We don't use styled components anymore and I saw the same styles in `TransactionDialog`. Adjust it the same way.